### ISSUE TITLE
No "Dump DICOM tags" context menu.

### ISSUE THREAD

**Issue by brenthuisman** [smikitky/vscode-dicom-dump#5]
Although originially my dicom files did not have any extension at all, adding .dcm or .dicom does not result in the context menu options. When I load the files, I get the default warning the file is not displayed because it's binary or has an unsupported encoding. Opening it anyway, or not, both don't result in the context options. Through dicom-parsers online demo it seems like there's nothing wrong with the files (all other tooling we have handles them fine too).

I restarted vsc and reinstalled the plugin a few times, no dice.

I use VSCodium, would that make a difference? It hasn't for any other plugin I use so far...

**Comment by smikitky** [comment]
Is your DICOM file on your local machine, and are you using an ordinary local workspace? If you're loading your DICOM files via one of `FileSystemProvider`s (e.g., [SSH FS](https://marketplace.visualstudio.com/items?itemName=Kelvin.vscode-sshfs)), this is a known issue. I have a plan to fix it (#4).

> I use VSCodium, would that make a difference?

I doubt it makes some difference, but I personally don't use it, so I may be missing something.

**Comment by brenthuisman** [comment]
All files are local.

**Comment by chafey** [comment]
I have the same issue - the context menu does not appear for files without any extension.  Can you make it work with any file regardless of extension?  Nice work BTW, I will use this :)

**Comment by smikitky** [comment]
@chafey Thank you for the suggestion. The visibility of the context menu items is [managed by VS Code itself](https://code.visualstudio.com/api/references/contribution-points#contributes.menus), and, last time I checked, I could control the visibility based only on the **file type**. Unfortunately, VS Code was (and still is) terribly bad at inferring the type of a file based on its contents if the file had no extension. I could remove the `when` condition altogether, but I did not want the DICOM dump menu to appear even on text files.

However, I noticed that the [when-clause contexts](https://code.visualstudio.com/docs/getstarted/keybindings#_when-clause-contexts) have been significantly expanded, so what you requested seems totally possible to me now. It's probably even possible to control the visibility of the dump menu according to the user preference.
